docs/BR.md

```diff
@@ -752,6 +754,17 @@ The Random Value SHALL remain valid for use in a confirming response for no more
 
 **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.
 
+Effective January 15, 2025:
+- When issuing Subscriber Certificates, the CA MUST NOT rely on Domain Contact information obtained using an HTTPS website, regardless of whether previously obtained information is within the allowed reuse period.
+- When obtaining Domain Contact information for a requested Domain Name the CA:
+     - if using the WHOIS protocol (RFC 3912), MUST query IANA's WHOIS server and follow referrals to the appropriate WHOIS server.
+     - if using the Registry Data Access Protocol (RFC 7482), MUST utilize IANA's bootstrap file to identify and query the correct RDAP server for the domain.
+     - MUST NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA that is more than 48 hours old, to ensure that it relies upon up-to-date and accurate information.
```

Comment:
We might want to allow for some caching in WHOIS, especially when following IANA referrers.

Say we query `whois.iana.org` for `cabforum.org`, this gives us a refer for the domain `ORG` to `whois.publicinterestregistry.org`:

```
% IANA WHOIS server
% for more information on IANA, visit http://www.iana.org
% This query returned 1 object

refer:        whois.publicinterestregistry.org

domain:       ORG
....
```

When you have multiple domains within the .org TLD that need validation (for a single order or several subsequent orders) and caching isn't permitted, you'll end up making the same query repeatedly, which can lead to hitting rate limits.

To align with RDAP we could make the following change:
```suggestion
     - MUST NOT rely on cached 1) WHOIS server information that is more than 48 hours old, or 2) RDAP bootstrap data from IANA that is more than 48 hours old, to ensure that it relies upon up-to-date and accurate information.
```

Reply:
Thanks for the suggestion @vanbroup. The idea was to permit caching for up to 48 hours for both WHOIS and RDAP. Accepting suggestion!